content/docs/04.5-devx/03-cluster-groups/35-ingress-cluster-group.md

```diff
@@ -0,0 +1,137 @@
+
+- If you are not using Palette's NGINX add-on profile, you can add the following to the profile you're using to reroute incoming requests from port 6443 to port 443. 
+
+    This will terminate TLS for your Apps at the cloud Load Balancer while still allowing you to connect to the API servers of your Virtual Clusters. Alternatively, you must use Cert Manager to issue certificates for App Ingress within the cluster.<br /><br />
+
+    ```
+    tcp:   
+    6443: "nginx/nginx-ingress-controller:443"  
+    # 8080: "default/example-tcp-svc:9000"
+    ```
+
+
+# Set Up Ingress
+
+1. Log in to Palette as **Tenant Admin**.
+<br />
+2. Identify each host cluster that requires the addition of an NGINX Ingress Controller profile.
+
+    This can be:
+
+    - All the host clusters in an existing Cluster Group, <br />
+        or
+    
+    - Existing host clusters that you will add to a new Cluster Group. <br /><br />
+
+3. Add the ```ingress-nginx-host-cluster``` add-on profile to each host cluster. <br />
+    a. From the **Main Menu**, choose **Clusters** and select a cluster.<br />
+    b. In the **Profile** tab, click **Add add-on profile (+)** and select ```ingress-nginx-host-cluster```. <br />
+    c. Confirm and save your changes.
+<br />
+4. For each host cluster with ingress deployed, follow these steps to open a web shell, identify the External-IP of the LoadBalancer Service, and copy the record you will need to create a canonical Name (CNAME) Domain Name System (DNS) record:
+
+    a. From the **Main Menu**, select a cluster. The cluster **Overview** tab displays. 
+
+    b. In the **Details** section beneath **Metrics**, click the **Connect** button next to the Kubernetes config file to open a web shell. 
+    
+    c. Invoke the following command to display the External-IP of the ```ingress-nginx-host-cluster``` LoadBalancer Service: <br /><br />
+
+    ```
+    kubectl -n nginx get service nginx-ingress-controller
+    ``` 
+
+    d. Copy the record to your clipboard or to a text file. You will use the External-IP address to create a CNAME DNS record.
+    <br />
+
+    e. Close the web shell.
+    <br />
+
+5. Use your DNS provider to create a wildcard CNAME record that maps to the External-IP for the NGINX Ingress Controller. Paste the External-IP you copied from the web shell to create the CNAME record.
+<br />
+<InfoBox>
+    The CNAME record is also known as the host cluster DNS pattern.
+</InfoBox> 
+<br />
+6. Copy the CNAME record to your clipboard.
+<br />
+
+7. Ensure you are in Palette's Cluster Mode, under the Tenant Admin scope, and select **Cluster Groups** in the **Main Menu**, then select the Cluster Group that requires ingress.<br /><br />
+    a. From the **Host Clusters** tab, select **Settings > Clusters**.   
+    b. Choose **Ingress** as the **Cluster endpoint type**. 
+    c. Paste the name of the wildcard CNAME record into the **Host DNS** field.
```

Comment:
@ritawatson  The markdown looks correct but when rendered the c bullet is on the same line ??
![CleanShot 2023-01-24 at 08 21 30](https://user-images.githubusercontent.com/29551334/214334299-da0cab70-355a-4df4-9673-0396f37f87c0.png)

Reply:
fixed